pytorch_lightning/utilities/cli.py

```diff
@@ -89,20 +89,24 @@ def __init__(
         parser: LightningArgumentParser,
         config: Union[Namespace, Dict[str, Any]],
         config_filename: str,
+        overwrite: bool = False,
     ) -> None:
         self.parser = parser
         self.config = config
         self.config_filename = config_filename
+        self.overwrite = overwrite
```

Comment:
```suggestion
        self.allow_overwrite = overwrite
```

Reply:
I prefer the current argument name, also matches the one upstream